src/Mollie/WC/Gateway/Abstract.php

```diff
@@ -1252,20 +1252,15 @@ protected function getPaymentMethodTitle($payment)
 	 *
 	 * @return string
 	 */
-	public function getReturnRedirectUrlForOrder( WC_Order $order ) {
-
-		// Get order ID in the correct way depending on WooCommerce version
-		if ( version_compare( WC_VERSION, '3.0', '<' ) ) {
-			$order_id = $order->id;
-		} else {
-			$order_id = $order->get_id();
-		}
-
-		Mollie_WC_Plugin::debug( __METHOD__ . " $order_id: Determine what the redirect URL in WooCommerce should be." );
+	public function getReturnRedirectUrlForOrder( WC_Order $order )
+    {
+        $order_id = wooCommerceOrderId($order);
+        $debugLine = __METHOD__ . " $order_id: Determine what the redirect URL in WooCommerce should be.";
```

Comment:
Could you wrap the variable within curly braces? `__METHOD__ . " {$order_id}: Determine what the redirect URL in WooCommerce should be."`